chef-config/lib/chef-config/path_helper.rb

```diff
@@ -295,5 +295,28 @@ def self.writable_sip_path?(path)
       ChefConfig.logger.error("Cannot write to a SIP Path on OS X 10.11+")
       false
     end
+
+    # Splits a string into an array of tokens as commands and arguments
+    #
+    # str = 'command with "some arguments"'
+    # split_args(str) => ["command", "with", "\"some arguments\""]
+    #
+    def self.split_args(line)
+      cmd_args = []
+      field = ""
+      line.scan(/\s*(?>([^\s\\"]+|"([^"]*)"|'([^']*)')|(\S))(\s|\z)?/m) do |word, within_dq, within_sq, esc, sep|
+
+        # Appand the string with Word & Escape Character
```

Comment:
Minor: Fix typo in comment (Appand -> Append)

Reply:
Sorry for the typo. Fixed in #7499